.github/dependabot.yml

```
@@ -2,13 +2,11 @@ version: 2
 updates:
   - package-ecosystem: pip
     directories:
-      - /utils/cq/requirements.txt
+      - /utils/cq/
+      - /
```

Not sure if you were aware of this one or didn't include it intentionally:

```suggestion
      - /
      - utils/ansible/ftest/
```

---

Overlooked, thanks!